File: fontes/c/micro/micro05.c
#include<stdio.h>

int main()
{
 char nome[50];
 int x;
 int h = 0;
 int m = 0;
 int op;

 for(x = 0; x < 5; x++)
 {
	printf("Digite o nome: ");
	scanf("%s", nome);
	printf("1 - Homem ou 2 - Mulher: ");
	scanf("%d", &op);
	switch(op)
	{
		case 1:
			h = h + 1;
			break;
		case 2:
			m = m + 1;
			break;
		default:
			printf("Sexo só pode ser Homem(1) ou Mulher(2)!\n");
			break;
	}
 }
 printf("Foram inseridos %d Homens.\n", h);
 printf("Foram inseridos %d Mulheres.\n", m);

 return 0;
}

File: fontes/pascal/micro/micro05.pas
program Micro05;
    
var 
    nome, sexo: CHAR;
	x, h, m: INTEGER;
begin

    for x:= 1 to 5 do 
        begin
            writeln('Digite o nome: ');
            readln(nome);
            writeln('Escreva H - Homem ou M - Mulher: ');
            readln(sexo);
            
            case (sexo) of
               'H' : h:=h+1;
               'M' : m:=m+1;
            else
               writeln('Sexo só pode ser H ou M!');
            end;
        end;
    
        writeln('Foram inseridos',h,' Homens');
        writeln('Foram inseridos',m,' Mulheres')
end.